src/lib/mongoBackend/mongoSubCache.cpp

```diff
@@ -1056,9 +1069,17 @@ void mongoSubCacheStatisticsGet(int* refreshes, int* inserts, int* removes, int*
   {
     while (cSubP != NULL)
     {
+      char msg[256];
+
       unsigned int bytesLeft = listSize - strlen(list);
 
-      if (strlen(cSubP->subscriptionId) + 2 > bytesLeft)
+#if UpdateBugTests
```

Comment:
I removed it in d0c923d (and I have already put it back in my personal copy :-))